@uportal/esco-content-menu/src/components/ContentMenu.vue

```diff
@@ -101,6 +97,9 @@ export default {
       validator: (value) => ['auto', 'always', 'never'].includes(value),
       default: 'auto',
     },
+    userOrgIdAttributeName: {type: String, default: 'ESCOSIRENCourant'},
+    userAllOrgsIdAttributeName: {type: String, default: 'ESCOSIREN'},
+    orgLogoUrlAttributeName: {type: String, default: 'ESCOStructureLogo'},
```

Comment:
On these properties I think about to purpose a deep attribute object composition, like that we will be able to get the value on any place of the object (as example for orgs I have a sub-level to obtains optional properties as a map), I mean I want to be able to set an attribute like "OtherAttributes.ESCOStructureLogo".
If you know some libraries that make the job to recompose the object path from a String you are welcome :wink:

Reply:
Something like: https://lodash.com/docs#get ?

Reply:
yes like that but smaller !

Reply:
Like https://www.npmjs.com/package/lodash.get?